### PR TITLE
Page/login

### PR DESCRIPTION
**Pull request by naremanhilles**

### CODE REVIEW

File: client/src/Components/Layouts/Login/index.jsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    this.setState({
+      [name]: value.trim()
+    });
+  };
+
+  handleClick = (e) => {
+    e.preventDefault();
+    const { userName, password } = this.state;
+    loginValidation
+      .validate({ userName, password })
```
Comment (engshorouq) [inline client/src/Components/Layouts/Login/index.jsx]: we don't need validation on the login when a username or password wrong  just show message `username or password is incoorect`

File: client/src/Components/Layouts/Login/index.jsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        <div className='login-box'>
+          <h2>تسجيل الدخول</h2>
+          <input type="text" name="userName" id="userName" placeholder="إسم المستخدم" onChange={this.handleChange} className='input' />
+          <input type="password" name="password" id="password" placeholder="كلمة المرور" onChange={this.handleChange} className='input' />
+          <input type="submit" value="تسجيل الدخول" onClick={this.handleClick} className='login-button' />
+        </div>
+      </div>
+    );
+  }
+}
```
Comment (engshorouq) [inline client/src/Components/Layouts/Login/index.jsx]: leave line here

File: client/src/Components/Layouts/Login/index.jsx [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+  render() {
+    return (
+      <div className='login-page'>
+        <h2 className='title'>زاجل</h2>
+        <div className='login-box'>
+          <h2>تسجيل الدخول</h2>
+          <input type="text" name="userName" id="userName" placeholder="إسم المستخدم" onChange={this.handleChange} className='input' />
+          <input type="password" name="password" id="password" placeholder="كلمة المرور" onChange={this.handleChange} className='input' />
+          <input type="submit" value="تسجيل الدخول" onClick={this.handleClick} className='login-button' />
```
Comment (engshorouq) [inline client/src/Components/Layouts/Login/index.jsx]: the name of classes must have like that 
`className="login"`
`className="login__title"`
`className="login__box"`
`className="login__box-input"`
`className="login__box-button"`

File: client/src/Components/helper/validationSchema.js [hunk 4]
```diff
@@ -0,0 +1,14 @@
+const yup = require('yup');
```
Comment (engshorouq) [inline client/src/Components/helper/validationSchema.js]: delete this file

File: client/package.json [hunk 5]
```diff
@@ -12,7 +13,8 @@
     "react-dom": "^16.8.6",
     "react-router-dom": "^5.0.0",
     "react-scripts": "3.0.1",
-    "sweetalert2": "^8.11.1"
+    "sweetalert2": "^8.11.1",
+    "yup": "^0.27.0"
```
Comment (engshorouq) [inline client/package.json]: make `npm uninstall yap`

File: client/src/App.jsx [hunk 6]
```diff
@@ -1,10 +1,12 @@
 import React from 'react';
 import './App.css';
+// import Login from './Components/Layouts/Login';
 import Navbar from './Components/CommonComponent/Navbar';
 
 function App() {
   return (
     <div className="App">
+      {/* <Login /> */}
```
Comment (hshahwan) [inline client/src/App.jsx]: delete comments line

File: client/src/Components/Layouts/Login/style.css [hunk 7]
```diff
@@ -0,0 +1,44 @@
+.login-page {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: #201F2B;
+
+}
+.title{
+  height: 180px;
```
Comment (hshahwan) [inline client/src/Components/Layouts/Login/style.css]: we agreed to use rem not px, so change it

File: client/src/Components/helper/validationSchema.js [hunk 8]
```diff
@@ -0,0 +1,14 @@
...
+
+// eslint-disable-next-line import/prefer-default-export
+export const loginValidation = yup.object().shape({
```
Comment (hshahwan) [inline client/src/Components/helper/validationSchema.js]: no need for validation in the front end

File: client/src/Components/Layouts/Login/style.css [hunk 9]
```diff
@@ -0,0 +1,47 @@
+.login {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: #201F2B;
+
```
Comment (hshahwan) [inline client/src/Components/Layouts/Login/style.css]: extra line

**Pull request merged**